react-components/src/components/RevealToolbar/LayersButton/LayersButton.tsx

```diff
@@ -31,11 +31,7 @@ export const LayersButton = ({
 
   return (
     <>
-      <SelectPanel
-        placement="right"
-        appendTo={'parent'}
```

Comment:
There is most likely a bug here in cogs. Ironically removing the appendTo parent actually causes it to append to parent (which seems to be the default behaviour). @shubhsk88 Has been notified, though we are struggling to reproduce it outside of the reveal react components lib.